.changeset/kind-cycles-explain.md

```diff
@@ -0,0 +1,25 @@
+---
+'@nhost/hasura-auth-js': minor
+'@nhost/nhost-js': minor
+---
+
+Deprecate Axios
+
+Axios will be replaced by cross-fetch in the near future.
```

Comment:
We are changing t
```suggestion
Axios will be replaced by cross-fetch in the near future. The signature of 2 methods is changing because we are making the API more flexible. 
```